tests/test_lab_dev/test_simulator.py

```diff
@@ -53,6 +58,6 @@ def test_run(self, path):
         res = Simulator().run(circuit)
         exp = vac12 >> d1 >> d2 >> d12 >> a1 >> n12
 
-        assert res == exp
+        assert np.isclose(res, exp)
```

Comment:
not super concerned, but do you know why this isn't exact anymore?

Reply:
it is exact to float precision (1e-18), it's just that one shouldn't compare floats with ==